src/analysis/output.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+/**
+ * This module defines a rich output format for cred scores, so that we can use
+ * it to drive UIs and data analysis.
+ */
+import type {Alias} from "../plugins/identity/alias";
+import type {EdgeWeight} from "../core/weights";
+import type {PluginDeclaration} from "./pluginDeclaration";
+
+// unenforced
+export type Integer = number;
+export type TimestampMs = Integer;
```

Comment:
We can use the TimestampMs type from the timestamp util.

Even if it would not be enforced if it came directly from `JSON.parse()`. As imo `JSON.parse()` should be treated as returning `mixed` rather than `any`.

Reply:
Done.